index.bs

```diff
@@ -459,14 +484,35 @@ The {{height}} attribute MUST return the height in <a lt=px value>CSS pixels</a>
 <a>Picture-in-Picture window</a> associated with {{pictureInPictureElement}} if
 the |state| is |opened|. Otherwise, it MUST return 0.
 
+The {{document}} attribute MUST return the {{Document}} displayed in the
+<a>Picture-in-Picture window</a> if the window is displaying <a>arbitrary content</a>
+
 When the size of the <a>Picture-in-Picture window</a> associated with
 {{pictureInPictureElement}} changes, the user agent MUST <a>queue a task</a> to
 <a>fire an event</a> with the name {{resize}} at {{pictureInPictureElement}}.
 
+## Dictionary <code>PictureInPictureOptions</code> ## {#dictionary-picture-in-picture-options}
+
+<xmp class="idl">
+dictionary PictureInPictureOptions {
+  required double aspectRatio;
```

Comment:
Shall we simply have `width` and `height` instead of `aspectRatio` as this will require web developers to calculate it based on `video.videoWidth` and `video.videoHeight`?

Reply:
Done